src/test/java/com/puppycrawl/tools/checkstyle/AbstractGuiTestSupport.java

```diff
@@ -0,0 +1,69 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2022 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle;
+
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+
+import java.awt.Component;
+import java.awt.Container;
+import java.awt.GraphicsEnvironment;
+
+import org.junit.jupiter.api.BeforeAll;
+
+/**
+ * Abstract base class for testing GUI components.
+ */
+public abstract class AbstractGuiTestSupport extends AbstractModuleTestSupport {
```

Comment:
I am not seeing why this needs to be ModuleTest support. I am not seeing any `verify` method calls or anything related with running a module.

Reply:
Done. Changed to `AbstractPathTestSupport`.